server/sonar-server/src/main/java/org/sonar/server/computation/issue/SourceLinesCache.java

```diff
@@ -69,7 +70,8 @@ void init(String fileUuid) {
   public String lineAuthor(int lineId) {
     loadIfNeeded();
     if (lineId <= authors.size()) {
-      return authors.get(lineId - 1);
+      String author = authors.get(lineId - 1);
```

Comment:
We should go deeper in this discussion with the whole team. Currently null is used in most of our models for unset fields. Empty string currently means "information exists and value is blank". That's not the case here. Issue has no author.
Changing that would require to update the whole stack.